### PR TITLE
WIPP ReadMe File

### PR DESCRIPTION
**Pull request by mmwilder26**
WIPP Readme file

### CODE REVIEW

**Comment by mmwilder26** [comment]
Hey @JAG-UK ... let me know your thoughts/changes in regards to the readme file...thanks.

File: archivist_samples/wipp/README.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# Waste Isolation Pilot Plant (WIPP) Sample
+
+Tracking the Nuclear Waste Management lifecycle is an important aspect to ensure that all safety protocols and procedures have been successfully executed.  Fragmented communication and manual checks can lead to honest mistakes and redundancy.  Digitizing the lifecycle and exposing data to the right parties at the right time can decrease honest mistakes and increase effective communication.  
+
+RKVST Continuous Assurance Hub offers a solution to fragmented communication and manual checks. Parties have near real-time access to data increasing seamless and effective communication and stakeholders can control the sharing of data ensuring one can view information that is relevant.  Policies, procedures and images can be included/attached thus reducing multiple checks and 
+providing persons with the most recent documentation.
+ 
+This sample shows how to quickly get started with integrating Nuclear Waste Management lifecycle with RKVST Continuous Assurance Hub.
```
Comment (JAG-UK) [inline archivist_samples/wipp/README.md:8]: "This sample uses publicly-available information about WIPP (the Waste Isolation Pilot Plant) how to quickly get started with integrating Nuclear Waste Management lifecycle with RKVST Continuous Assurance Hub."

File: archivist_samples/wipp/README.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
+# Waste Isolation Pilot Plant (WIPP) Sample
```
Comment (JAG-UK) [inline archivist_samples/wipp/README.md:1]: Nuclear Waste Handling Lifecycle Sample

File: archivist_samples/wipp/README.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+* Python 3.6 and later versions are supported.
+
+* Install the [RKVST samples Python package](https://pypi.org/project/jitsuin-archivist-samples/ "PyPi package page")
+
+* Get an authorization bearer token and store it in the file `credentials/.auth_token`. If you don't know how to do this, please refer to the [RKVST documentation](https://docs.jitsuin.com/docs/setup-and-administration/getting-access-tokens-using-client-secret/ "Getting an auth token"). Make sure that the `credentials` folder is suitably restricted by disallowing root and group access.
+
+
+## Running the sample
+
+The sample creates a WIPP object and uploads images, policies and procedures relevant to certain events.  The events simulate Nuclear Waste Disposal lifecycle from waste characterization through emplacement. Thus providing a "Continuous Assurance" example of Nuclear Waste Managment.
```
Comment (JAG-UK) [inline archivist_samples/wipp/README.md:22]: "creates a WIPP object" -> "registers nuclear waste assets"

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: archivist_samples/wipp/README.md (hunk 1 above)
Comment (mmwilder26) [reply archivist_samples/wipp/README.md]: Updated

File: archivist_samples/wipp/README.md (hunk 2 above)
Comment (mmwilder26) [reply archivist_samples/wipp/README.md]: Updated

File: archivist_samples/wipp/README.md (hunk 3 above)
Comment (mmwilder26) [reply archivist_samples/wipp/README.md]: Updated

**Review by mmwilder26 — COMMENTED** [review]
Hey @JAG-UK   . . . made the suggested changes.

**Review by JAG-UK — APPROVED** [review]

**Comment by mmwilder26** [comment]
thank you @JAG-UK

**Pull request merged**